examples/NetworkAnalysis/Transmitter/Transmitter_esp8266

```diff
@@ -0,0 +1,76 @@
+// a 10K pull-down resistor is required
+// place it between the GND and the communications pin
+// if you are getting a report of every message in Busy bucket, add a resistor
+
+#if defined(ESP8266)
```

Comment:
@aperepel Why you add this condition and inclusion?
With nodeMCU I generally don't have to include ESP8266.h (or is automatically included)